frontend/assets/locales/en-US.json

```diff
@@ -109,6 +109,26 @@
                 "settings": "Settings",
                 "logout": "Logout",
                 "login": "Login"
+            },
+            "questionnaire": {
+                "responses": "Responses",
+                "message": "Message",
+                "searchbar": "Search bar...",
+                "questions": {
+                    "title": "Questions",
+                    "unlock": "(un)lock",
+                    "warning": "Warning: this affects all hackers...",
+                    "defaultQuestions": "Default questions",
+                    "optionalQuestions": "Optional questions"
+                }
+            },
+            "message": {
+                "title": "Send a message",
+                "subject": "Subject",
+                "send": "Send"
+            },
+            "editApplicant": {
+                "title": "Edit"
             }
```

Comment:
Can you move these inside "pages.questionnaire"? I've tried to next subpages inside their parents' locale objects (e.g. "accessibility" is a child of "pages.application", not a sibling).